events-api/src/modules/events/events.repository.ts

```diff
@@ -16,6 +20,130 @@ export class EventsRepository {
             events
     `);
 
-    return result;
-  }
+        return result;
+    }
+
+    static createEvent(event: EventsCreateDto): Promise<EventModel[]> {
```

Comment:
Why to return an array?

Reply:
fixed, only single records are now returned when needed